tools/wltests/test_series

```diff
@@ -893,6 +893,11 @@ test_sha1() {
 	echo
 	c_info "Check current kernel..."
 	if device_in_adb; then
+		export ANDROID_OS_VERSION=${ANDROID_OS_VERSION:-$($ADB shell getprop ro.build.version.release)}
```

Comment:
These are horribly long lines... what about these?... which should be slightly shorter:

```bash
[ ! -z $ANDROID_OS_VERSION ] || \
    export ANDROID_OS_VERSION="$($ADB shell getprop ro.build.version.release)"
[ ! -z $ANDROID_OS_PATCH_LEVEL ] || \
    export ANDROID_OS_PATCH_LEVEL="$($ADB shell getprop ro.build.version.security_patch)"
```